wasm-runtime/polkadot/src/runtime/session.rs

```diff
@@ -54,47 +55,49 @@ pub fn set_validators(new: &[AccountID]) {
 	consensus::set_authorities(new);
 }
 
+/// Hook to be called after transaction processing.
+pub fn check_rotate_session() {
```

Comment:
So does it `check` or actually `rotate`? `check` implies no modifications to me.

Reply:
it checks to see whether it should rotate it and rotate it if so. i'd call something that only evaluates whether it should rotate something like `should_rotate`...